src/Storefront/Page/GenericPageLoader.php

```diff
@@ -94,7 +94,7 @@ public function load(Request $request, SalesChannelContext $context): Page
         });
     }
 
-    public function getSystemConfig(string $config, string $salesChannelId): string
+    private function getSystemConfig(string $config, string $salesChannelId): string
```

Comment:
Thanks! SO YES YES!

Reply:
Why in general add a new method, in my personal opinion this is an not needed abstraction :-)